hack/update-generated-clientsets.sh

```diff
@@ -28,7 +28,7 @@ function generate_clientset_for() {
   grouppkg=$(realpath --canonicalize-missing --relative-to=$(pwd) ${package}/..)
   client-gen --clientset-path="${grouppkg}/generated" \
              --input-base="${package}"                            \
-             --output-base="../../.."                           \
+             --output-base="${GOPATH}/src"                            \
```

Comment:
How does the tool use this value? Where should `--output-base` point to? The root of what?